lib/Php/ClassGenerator.php

```diff
@@ -37,23 +98,52 @@ private function handleBody(Generator\ClassGenerator $class, PHPClass $type)
         return true;
     }
 
+    /**
+     * @param PHPClass $class
+     * @return bool
+     */
     private function isNativeType(PHPClass $class)
     {
-        return ! $class->getNamespace() && in_array($class->getName(), [
-            'string',
-            'int',
-            'float',
-            'integer',
-            'boolean',
-            'array',
-            'mixed',
-            'callable'
-        ]);
+        return !$class->getNamespace() && in_array(
+            $class->getName(),
+            [
+                'string',
+                'int',
+                'float',
+                'integer',
+                'boolean',
+                'array',
+                'mixed',
+                'callable'
+            ]
+        );
+    }
+
+    /**
+     * @param string $shortDescription
+     * @param string $longDescription
+     * @param array $tags
+     * @return DocBlockGenerator
+     */
+    private function createDocBlockGenerator($shortDescription = null, $longDescription = null, array $tags = [])
+    {
+        $docBlockGenerator = $this->docBlockGeneratorPrototype ?
```

Comment:
`clone` is always a risky feature. i will prefer to move here the code from the generator:

```php

            $docBlockGenerator = new DocBlockGenerator();
            $docBlockGenerator->setWordWrap(false);
```           

Doing this you do not need `setDocBlockGeneratorPrototype` anymore